.gitignore

```diff
@@ -1 +0,0 @@
-node_modules
```

Comment:
- [ ] Please, never delete this important file `.gitignore`, always try to keep this file and having `node_modules` inside 😉

Reply:
Will do! Thanks a lot, Dante for the feedback. Cheers!

Reply:
- This has been resolved.